src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -166,6 +166,8 @@ public void AddToStatusDescription(string message)
             _connectionStatsIdForConnection = Interlocked.Increment(ref _connectionStatsId);
 
             CurrentBatchId = NonExistentBatch;
+
+            _buffer.Capacity = BufferCapacityInBytes;
```

Comment:
you do not need to set it here, this will grow it  to 2mb even if 100kb would be max